journalpage/templates/journalpage/about.html

```diff
@@ -211,7 +211,17 @@ <h5 id="item-1-8">{% trans 'Patrocinadores e agências de Fomento' %}</h5>
                         <h5 id="item-1-9">{% trans 'Preservação digital' %}</h5>
                         <ul type="disc">
                             {% for dp in digital_preservation %}
-                                <li>{{ dp.rich_text|richtext }}</li>
+                                <li>{{ dp.name }}</li>
+                                {% if dp.acronym == 'CLOCKSS' %}
+                                  <!-- CLOCKSS system has permission to ingest, preserve, and serve this Archival Unit -->
+                                  <div class="widget widget-Lockss widget-instance-IssueBrowseByYear_Lockss">
+
+                                    <input class="at-locks-messaging" type="hidden" value="LOCKSS system has permission to collect, preserve, and serve this Archival Unit" />
+                                  
+                                    <input class="at-clockss-messaging" type="hidden" value="CLOCKSS system has permission to ingest, preserve, and serve this Archival Unit" />
+                                  
+                                  </div>
+                                {% endif%}
```

Comment:
@samuelveigarangel No lugar de realizar a verficação dentro do `for`, fazer fora do `for`, consultando uma variável se é ou não CLOCKSS. Esta variável deve ser criada no contexto da view.